content/menu.md

```diff
@@ -510,6 +510,8 @@ site_menu:
     url: /knowledge-base/tutorials/session-state
   - category: Knowledge base / Tutorials / Build conversational apps
     url: /knowledge-base/tutorials/build-conversational-apps
+  - category: Knowledge base / Tutorials / LLM tutorial
+    url: /knowledge-base/tutorials/llm-tutorials
```

Comment:
`LLM tutorial` title here is inconsistent with what's in the file above currently (`LLM Tutorials`).

As noted above I think a clearer title ("Getting started with LLMs" or similar) would be better

Reply:
I'll go with the shorter "LLM quickstart" you suggested. I think phrases are generally harder to read when skimming and as much as possible I try not to "bury the lead." I think LLM as the first word will make it more visible within navigation.